BICCN-policies/dataset-metadata.yaml

```diff
@@ -0,0 +1,85 @@
+identifier: REQUIRED ## Post upload (or during dandi organize)
+name: REQUIRED
+description: REQUIRED
+contributors: # required for author and contact
+- orcid: # REQUIRED
```

Comment:
One comment - PIs likely to have an ORCID, but the other contributors may not have them such as the lead RA who generated the data. What if that lead RA has left the lab and can't be convinced to create an ORCID? 
ORCIDs seem like they would be good, but I can see cases where we won't have them, because they have to be created by the individual - unless someone knows of a way around that?

Reply:
for the moment we are going with orcid, but will allow people to add name/email/affiliation.